# RobotInterInfo.msg

int8[] inter_1
int8[] inter_2
int8[] inter_3
int8[] inter_4
int8[] inter_5